Makefile

```diff
@@ -1,33 +1,51 @@
 ROOT = $(shell git rev-parse --show-toplevel)
 
-build:
-	./scripts/build.sh -o $(ROOT)
+# builds
 
-release-build:
-	./scripts/build.sh -o $(ROOT) -r
+youki: youki-dev # helper
 
-test-all: test oci-integration-test integration-test features-test
+youki-dev:
+	./scripts/build.sh -o $(ROOT) -c youki
 
-test: build
-	cd crates && cargo test
+youki-release:
```

Comment:
I think most people expect the `build` subcommand and it builds youki itself. Why not have an alias as a helper? I'd rather just have `build`.
How about using an environment variable to switch between release and non-release?

Reply:
Would it be fine to have build similar to how youki is here? that way `make build` , `make youki` and `make youki-dev` would do the same thing. Alternatively, we can alias `build` to `youki-release` as well. WDYT?